Client/src/Components/Button/index.jsx

```diff
@@ -48,10 +48,11 @@ const levelConfig = {
  * <Button type="submit" level="error" label="Error" disabled sx={{marginTop: "1rem"}}/>
  */
 
-const Button = ({ type, level, label, disabled, img, onClick, sx }) => {
+const Button = ({ id, type, level, label, disabled, img, onClick, sx }) => {
   const { variant, color } = levelConfig[level];
```

Comment:
Should we have a default value here? just incase variant and color become undefined if level is not found?